### PR TITLE
(DOCSP-39191) VPC Private Endpoint note about IP Access Lists

### PR DESCRIPTION
**Pull request by krollins-mdb**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-39191

- [Use a VPC Private Endpoint](https://preview-mongodbkrollinsmdb.gatsbyjs.io/atlas-app-services/DOCSP-39191/security/private-endpoints/#ip-access-lists-and-private-endpoints)

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [x] Did you tag pages appropriately?
  - genre
  - programming_language
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [x] Create a Jira ticket for related docs-realm work, if any

### Release Notes

- **Secure Your App**
  - Use a VPC Private Endpoint: Add VPC Private Endpoint note about IP Access Lists

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/security/private-endpoints.txt [hunk 1]
```diff
@@ -372,6 +372,16 @@ You can toggle this setting in three ways:
   <operation/allNonVpcClientRequests>` with a boolean value (``true`` to remove
   the default restriction, or ``false`` to reinstate it.)
 
+IP Access Lists and Private Endpoints
+-------------------------------------
+
+When you enable private endpoints, you can still enable access to your Atlas
+clusters using other methods, such as adding public IPs to :ref:`IP access lists 
+<ip-access-list>`.
```
Comment (melodeeli98) [inline source/security/private-endpoints.txt:380]:
```suggestion
When you enable private endpoints, you can still enable access to your App Services application using other methods, such as adding public IPs to :ref:`IP access lists 
<ip-access-list>`.
```

File: source/security/private-endpoints.txt [hunk 2]
```diff
@@ -372,6 +372,16 @@ You can toggle this setting in three ways:
...
+
+Private endpoints do not need to be added to IP access lists, as requests
```
Comment (melodeeli98) [inline source/security/private-endpoints.txt:382]:
```suggestion
Private IPs do not need to be added to IP access lists, as requests
```

File: source/security/private-endpoints.txt [hunk 3]
```diff
@@ -372,6 +372,15 @@ You can toggle this setting in three ways:
   <operation/allNonVpcClientRequests>` with a boolean value (``true`` to remove
   the default restriction, or ``false`` to reinstate it.)
 
+IP Access Lists and Private Endpoints
```
Comment (dacharyc) [inline source/security/private-endpoints.txt:375]: If I'm reading this right - that Private Endpoints ignore the IP Access List - it feels a bit like we're burying the lede here. I might reframe/rename this section to something like:

"Private Endpoints Ignore IP Access Lists" and lead with that detail. Then, you could follow with the information that public requests still adhere to the IP Access List as expected.

File: source/security/private-endpoints.txt [hunk 4]
```diff
@@ -372,6 +372,15 @@ You can toggle this setting in three ways:
...
+-------------------------------------
+
+When you enable private endpoints, you can still enable access to your App Services application using other methods, such as adding public IPs to :ref:`IP access lists
```
Comment (dacharyc) [inline source/security/private-endpoints.txt:378]: Nit - this line should be wrapped as it looks significantly longer than our 80 character line width.

**Review by dacharyc — APPROVED** [review]
If I've correctly understood what we're trying to communicate here, I'd suggest reframing this note. But my recommendation is subjective, and/or I could have misunderstood what we're communicating, so it's a non-blocking suggestion.

File: source/security/private-endpoints.txt (hunk 4 above)
Comment (krollins-mdb) [reply source/security/private-endpoints.txt:378]: Ah, yep. Thank you! I committed one of Melodee's suggestions and didn't think the break the line.

File: source/security/private-endpoints.txt (hunk 3 above)
Comment (krollins-mdb) [reply source/security/private-endpoints.txt:375]: That's definitely clearer. Thank you!

**Pull request merged**